pkg/metadata/inventories/inventories.go

```diff
@@ -285,4 +286,7 @@ func initializeConfig(cfg config.Config) {
 	SetAgentMetadata(AgentLogsEnabled, config.Datadog.GetBool("logs_enabled"))
 	SetAgentMetadata(AgentCSPMEnabled, config.Datadog.GetBool("compliance_config.enabled"))
 	SetAgentMetadata(AgentAPMEnabled, config.Datadog.GetBool("apm_config.enabled"))
+	// NOTE: until otlp config stabilizes, we set AgentOTLPEnabled in cmd/agent/app/run.go
+	// Also note we can't import OTLP here, as it would trigger an import loop - if we see another
+	// case like that, we should introduce a register-a-callback mechanism to handle these cases.
```

Comment:
We should rather move `IsEnabled` to `pkg/config/otlp` so it is available here without import loops or callback machinery.

Reply:
Good point, I'll look into doing that - it should make things much easier.

Reply:
Ok, so I tried doing this and after 2 hours I think this will be pretty hard and require its own PR. If you're ok with that, I'll fix your previous comment on the `cmd/agent/app/run.go` file to get this merged and discuss the best approach to the config refactor with the rest of our team (it will require shuffling a lot of code around to make it work).

Reply:
Yes that's fine. Do you think it would make sense to at least update the comment? Even if a refactor is biggish, it might still be preferable to adding callbacks.